syntax = "proto2";
import "nanopb.proto";

message ci_register_pb
{
	enum Log {
		LOG = 0;
		EXCLUDE_LOG = 1;
	}
	enum Step {
		STEP_1 = 0;
		STEP_2 = 1;
	}
	required uint32 msgID = 1;
	required uint32 cmdCode = 2;
	required Step step = 3 [default = STEP_1];
	required Log log = 4 [default = EXCLUDE_LOG];
}
